public/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="main.css" />
+  <script src="https://kit.fontawesome.com/9a9c81b339.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
+    integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
+    crossorigin="" />
+  <script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"
+    integrity="sha512-gZwIG9x3wUXg2hdXF6+rVkLF/0Vi9U8D2Ntg4Ga5I5BZpVkVxlJWbSQtXPSiUTtC0TjtGOmxa1AJPuV0CPthew=="
+    crossorigin=""></script>
+</head>
+
+<body>
+  <!--  Header section  -->
+  <div id="header">
+    <h1>Yelpington</h1>
+  </div>
+
+  <!--  center section for sidebar and map  -->
+  <div class="middleContainer">
+    <!--  Map  -->
+    <div id='map'></div>
+    <!--  SideBar  -->
+    <nav>
+      <ol id='idList'></ol> <!--  Ordered list to recieve list items for side bar  -->
+      <script src="main.js"></script> <!--  connect to javascript file  -->
```

Comment:
This should be at the bottom of your file, just above the script tag where you call getData()